### PR TITLE
Morework

### PR DESCRIPTION
**Pull request by AnointingMax**

### CODE REVIEW

**Review by danieliheonu — APPROVED** [review]

**Review by LeanKhan — COMMENTED** [review]
Nice one 👍

**Review by LeanKhan — COMMENTED** [review]
In line 394, where you define the schedule_job function, will it be defined by the time you are calling it after an Election is created?

**Comment by danieliheonu** [comment]
In line 394, where you define the schedule_job function, will it be defined by the time you are calling it after an Election is created?

File: app/routes.py [hunk 1]
```diff
@@ -71,6 +71,7 @@ def create_election():
 
     form = ElectionForm()
     if form.validate_on_submit():
+        # is this necessary
```
Comment (LeanKhan) [inline app/routes.py:74]: no

**Review by LeanKhan — APPROVED** [review]
Everything looks good and works 👍 well done

**Pull request merged**